starfish/pipeline/features/spots/detector/gaussian.py

```diff
@@ -100,6 +112,9 @@ def fit(self, blobs_image):
         fitted_blobs['y_min'] = np.clip(np.floor(fitted_blobs.y - fitted_blobs.r), 0, None)
         fitted_blobs['y_max'] = np.clip(np.ceil(fitted_blobs.y + fitted_blobs.r), None, blobs_image.shape[1])
 
+        # location values must be >= 0
+        fitted_blobs[fitted_blobs < 0] = 0
```

Comment:
What does this mean?  What's the operation that happens when one compares a table to a value?

Reply:
`fitted_blobs < 0` will produce a `table_of_booleans`. 
`fitted_blobs[table_of_booleans] = 0` will set any value that evaluated to `True` equal to zero. 

Overall, it's executing a floor operation at zero.

Reply:
I don't think this is necessary any more, since we clip the x_min,x_max,y_min,y_max columns.

Reply:
Ah, true. I didn't notice that change when I rebased. Will fix.